### ISSUE TITLE
Eliminate extractors for deceased websites

### ISSUE THREAD

**Issue by alerikaisattera** [ytdl-org/youtube-dl#29241]
Certain websites supported by youtube-dl (for instance, liveleak) have been closed. I suggest removing extractors for these websites, for they don't serve any purpose anymore.

**Comment by pukkandan** [comment]
You can't expect the maintainers to periodically go through the massive list of supported websites and check which are alive. I suggest you open an issue (or even better, a PR) instead when you find a dead website

**Comment by rautamiekka** [comment]
No need to go check the sites manually, it can be scripted (then manually googled for a little. Not a huge amount of work, but ytdl devs are already overly busy), even just receiving the HTTP response code is enough.

**Comment by alerikaisattera** [comment]
> You can't expect the maintainers to periodically go through the massive list of supported websites and check which are alive. I suggest you open an issue (or even better, a PR) instead when you find a dead website

Fair point. Maybe I should try to check the supported sites list myself to see which ones are dead

**Comment by alerikaisattera** [comment]
> No need to go check the sites manually, it can be scripted (then manually googled for a little. Not a huge amount of work, but ytdl devs are already overly busy), even just receiving the HTTP response code is enough.

Nhis may be a good idea, but it would fail to distinguish a truly dead website from just a temporarily down one

**Comment by pukkandan** [comment]
Yes, @rautamiekka 's suggestion can help narrow down the potential cases, but at the end, a manual review of each website is needed

**Comment by rautamiekka** [comment]
> > No need to go check the sites manually, it can be scripted (then manually googled for a little. Not a huge amount of work, but ytdl devs are already overly busy), even just receiving the HTTP response code is enough.
> 
> This may be a good idea, but it would fail to distinguish a truly dead website from just a temporarily down one

Ya, it can't tell a temporarily dead one, but it does help to narrow them down massively.

With the temporary death in mind, the script could be extended to warn after, say, 5 failures over, say, 5 weeks (1 check/failure a week). If the site's still returning the same response after the a bit over a month of time it's quite likely staying down.
